### PR TITLE
Hello Microverse Project PR

### PR DESCRIPTION
**Pull request by diarisdiakite**
Added:
- index.html
- css/styles.css
- .github/workflows/linters.yml

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (Kossi-stack) [inline .gitignore:2]: - [x] Kindly add test.md file.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @diarisdiakite,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [x] Kindly edit the PR's title to something more descriptive. it can be "Hello Microverse Project PR or Hello Microverse Project".
 - [x] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
 - [x] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 - [x] Kindly fix the conflict you have in the branch
Screenshot 2023-03-21 at 19 39 26
 
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .gitignore (hunk 1 above)
Comment (diarisdiakite) [reply .gitignore:2]: Thanks I did it @Kossi-stack

**Comment by diarisdiakite** [comment]
## Hello-Microverse PR
In this PR, I:
###  Added the following files: 
- index.html
- css/styles.css]
- linters.yml
- .gitignore, 
- package.json and .
- hintrc
- stylelintrc.json;
- Readme.md
- test.md (ignored by git)

### Fixed the lighthouse issue not finding chrome

### Resolved the readme.md conflict

File: README.md [hunk 2]
```diff
@@ -1,2 +1,103 @@
...
+
+## 💻 Getting Started
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+
+
+### Setup
+Clone this repository to your desired folder:
```
Comment (braincee) [inline README.md:29]: - [ ] [OPTIONAL] Kindly add setup instructions for developers to clone the project.
Example: `https://github.com/diarisdiakite/Hello-Microverse-1.git `

File: README.md [hunk 3]
```diff
@@ -1,2 +1,103 @@
...
+[Hello-Microverse] key features are:
+
+[create-a-new-project-from-templates]
+[Add-new-collaborators]
+[Add-new-templates-for-projects]
+(back to top)
+
+## Live Demo link
+[Hello-Microverse](https://github.com/diarisdiakite/Hello-Microverse/pull/1)
+(back to top)
```
Comment (braincee) [inline README.md]: - [ ] [OPTIONAL] Please I suggest you remove the live demo link since this project is not deployed

File: README.md [hunk 4]
```diff
@@ -1,2 +1,103 @@
...
+
+## 💻 Getting Started
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+
```
Comment (braincee) [inline README.md]: - [ ] [OPTIONAL] Please few bulletins to the prerequisites section
Example:
- Have some knowledge of HTML and CSS
- Be familiar with git commands
- Basics understanding of Linters

File: README.md [hunk 5]
```diff
@@ -1,2 +1,103 @@
...
+### Setup
+Clone this repository to your desired folder:
+
+### Install
+Install this project with: [npm](https://www.npmjs.com/)
+
+```[npm]
+npm (--npm-command-hello-microverse/app)
+```
```
Comment (braincee) [inline README.md]: - [ ] [OPTIONAL] Please add the required installation process
Example: `npm install`

**Review by braincee — APPROVED** [review]
Hi @diarisdiakite  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 4 above)
Comment (diarisdiakite) [reply README.md]: Well noted @braincee

File: README.md (hunk 5 above)
Comment (diarisdiakite) [reply README.md]: OK @braincee, well noted. I was just thinking that it might need an entire command set beforehand.

File: README.md (hunk 2 above)
Comment (diarisdiakite) [reply README.md:29]: Done!!!
Thanks

**Pull request merged**